components/raftstore/src/store/fsm/store.rs

```diff
@@ -1172,12 +1177,13 @@ impl<EK: KvEngine, ER: RaftEngine> RaftBatchSystem<EK, ER> {
         &mut self,
         meta: metapb::Store,
         cfg: Arc<VersionTrack<Config>>,
+        cop_cfg: CopConfig,
```

Comment:
The `CopConfig` may become stale after a dynamic config change, to get the update-to-date `CopConfig` we may need to pass down the `ConfigController` from `cmd/src/server.rs` and read config value from it.